src/CMakeLists.txt

```diff
@@ -548,10 +544,8 @@ if(BUILD_CLIENT)
 	target_link_libraries(
 		${PROJECT_NAME}
 		${ZLIB_LIBRARIES}
-		${IRRLICHT_LIBRARY}
-		${JPEG_LIBRARIES}
+		IrrlichtMt::IrrlichtMt
 		${BZIP2_LIBRARIES}
-		${PNG_LIBRARIES}
 		${X11_LIBRARIES}
```

Comment:
x11 + bzip should also disappear

Reply:
Removing X11 results in an undefined symbol in client/renderingengine.cpp